router/start.sh

```diff
@@ -243,6 +277,9 @@ function start_dnsmasq {
 
 function main {
 	check_configuration
+	if [ -z ${TUNNEL_MTU+x} ]; then
+		set_tunnel_mtu
```

Comment:
We only want to change the tunnel MTU for 6in4, not OpenVPN (because OpenVPN runs on top of UDP which takes advantage of the system's MTU detection), so I suggest moving this call (including `if` clause) a few lines below, inside the block that tests `if [ "$TUNNEL_TYPE" = 'he-6in4' ]`, something like:
```sh
	if [ "$TUNNEL_TYPE" = 'he-6in4' ]; then
		if [ -z "${TUNNEL_MTU}" ]; then
			set_tunnel_mtu
		fi
		del_6in4_tunnel
		setup_6in4_tunnel
	fi
```